### ISSUE TITLE
Gather traffic data for service endpoints

### ISSUE THREAD

**Issue by ewilson-adhoc** [department-of-veterans-affairs/va.gov-team#44045]
## Story
As a vets-api development team I would like to understand the potential traffic volumes that future microservices might have, so that I can take this into consideration when I prioritize them. I would like to know what the granularity of the traffic volumes that are available to me, e.g. historically, how far back can the data be captured and in what granularity (minutes, days, weeks, months, etc.).

## Acceptance Criteria 
- [ ] Traffic volume data from the different services is added to the service page.
- [x] Create a grafana dashboard to display this information.
- [x] This will be accessible on confluence and added to [this page](https://vfs.atlassian.net/wiki/spaces/IET/pages/2241822742/Proposed+Services)

**Comment by ewilson-adhoc** [comment]
Before we can point this story, we need to determine if there is a better way to get data other than from Loki. I will open a separate story from that.

**Comment by considerable** [comment]
Last endpoints research is in this [google doc](https://docs.google.com/spreadsheets/d/14iNeWOyaMe0aqVtgfCqTE7cl3xAfQ7YJDsjRbADOXCk/edit#gid=1482742543)

**Comment by MrMorie** [comment]
https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/READMES/grafana-dashboard-cms.md

**Comment by MrMorie** [comment]
https://depo-platform-documentation.scrollhelp.site/developer-docs/centralized-logging-with-loki-and-grafana

**Comment by MrMorie** [comment]
Created a new Grafana dashboard with a single widget at the moment. When the time is right we may move the newly created widget to the previously existing Vets-API Dashboard.

New Dashboard: https://grafana.vfs.va.gov/d/iPlfSzkVk/va-pst-iet?editPanel=2&orgId=1&from=now-24h&to=now 

Existing Vets-API dashboard for reference: https://grafana.vfs.va.gov/d/000000017/vets-api?orgId=1

**Comment by FHiatt** [comment]
add a few sentences tried doing via Loki, e.g.  the limitation is that we can only go back a month

Make reference to a need for standardized logging and metrics collection for common stats. Make reference to we are using Grafana but using Datadog might provide a better method for collecting this data (add small rationale or exaplanation of why).

**Comment by MrMorie** [comment]
We attempted to produce traffic data using Loki as a data source, unfortunately generating the requested dashboards using Loki queries was difficult and we are not able to generate them within the time box. 

It does appear that loki logs do go as far back as September 2021, however actually generating graphs/queries was not possible this sprint.

We were able to generate traffic data easily (and looks accurate) using Prometheus metrics. The downside of using these metrics is the inability to query further back than a month.
